build/checkstyle.xml

```diff
@@ -48,6 +48,9 @@
     <module name="EmptyStatement"/>
     <module name="SuperFinalize"/>
     <module name="EqualsHashCode"/>
+    <module name="EmptyCatchBlock">
+      <property name="exceptionVariableName" value="ignore|ignored"/>
```

Comment:
This is the default rule for Intellij as well. It stops complaining about empty catch block when parameter name is `ignore` or `ignored`. There were no conflicts with other variables when I've fixed the code :)